README.md

```diff
@@ -1,32 +1,49 @@
-# f1-2021-analysis
+# Visualising the Formula 1 2021 Season
 
+## Welcome!
 
-## Contributors and Maintainers
+Whether you consider yourself a true Formula 1 veteran, or this is th first time you've heard of the sport, we bid you welcome to our dashboard! Firstly a quick overview of what you can find on this page:
+
+- [Visualising the Formula 1 2021 Season](#visualising-the-formula-1-2021-season)
+  - [Welcome!](#welcome)
+  - [The Contributors](#the-contributors)
+  - [The Motivation](#the-motivation)
+  - [The Description](#the-description)
+  - [The Contributing Guidelines](#the-contributing-guidelines)
+  - [The License](#the-license)
+  - [The Credits](#the-credits)
+
+![Lights out](/www/sketch/lights_out.png)
+
+## The Contributors
+
+We are four students at UBC, Vancouver, part of the Master of Data Science program:
 
 - [Tanmay Agarwal](https://github.com/tanmayag97)
 - [Julie Song](https://github.com/YXIN15)
 - [Eric Tsai](https://github.com/erictsai1208)
 - [Renzo Wijngaarden](https://github.com/RenzoWijn)
 
-## Description
+## The Motivation
+
+The 2021 Formula 1 season was one of the most exciting in the history of the sport. A title fight in the drivers championship that left both the neutral viewer and fans of either Red Bull's Verstappen and Mercedes' Hamilton on the edge of their seat and beyond until the very last lap of the year in Abu Dhabi, and an equally as engaging brawl in the constructors midfield makes no one deny its legendary status.  
+
+With all these developments throughout the season's calendar it can be hard to remember how it all went down. We are looking to provide a helping hand in reliving this awesome year, by summarising and visualising all the important statistics of 2021!
```

Comment:
We could also explicitly link the dashboard here: https://eric1208.shinyapps.io/f1-2021-analysis/